app/views/api/v1/transactions/_transaction.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.id transaction.id
```

Comment:
@yunixon yo my write it shorter using `json.(...)`. If you do it only for amount then just write amount separately.
And I think it's not good idea to use `to_s`. Use money_rails gem to convert to currency representation, or render `amount_cents` so client may convert and render as he wish.